### PR TITLE
license info updated

### PR DESCRIPTION
**Pull request by mhabimana**
license info updated

### CODE REVIEW

**Comment by mhabimana** [comment]
@isedwards - Don't review and merge this pull request. I was showing (demonstrating to ) Tsitsi how to go about via Skype screen sharing to enable her  start the task to update the license info

File: ClimsoftVer4/ClimsoftVer4/formImports.vb [hunk 1]
```diff
@@ -1,4 +1,19 @@
-﻿Public Class formImports
+﻿' CLIMSOFT - Climate Database Management System
+' Copyright (C) 2015
```
Comment (isedwards) [inline ClimsoftVer4/ClimsoftVer4/formImports.vb]: @tmundoga, @mhabimana - when you update the licence information could you update the Copyright year from `2015` to `2017`?

Comment (tmundoga) [inline ClimsoftVer4/ClimsoftVer4/formImports.vb]: I will update the copyright year also.

Comment (mhabimana) [inline ClimsoftVer4/ClimsoftVer4/formImports.vb]: @tmundoga  Thanks

**Pull request closed without merging**